setup.py

```diff
@@ -46,7 +46,7 @@ def get_extras_require() -> str:
             "doc8",
             "scipy",
             "pillow",
-            "pettingzoo>=1.12",
+            "pettingzoo>=1.17",
```

Comment:
Any reason?

Reply:
just good to keep the dependencies up-to-date in general